site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-list.vm

```diff
@@ -445,6 +445,13 @@
 				</div>
 			#end
 		</div>
+		<div>
+			<h4>
```

Comment:
Rather than an h4 here, I'd prefer this message to be in an information box:

`<div class="messageInformation">
#if ($!viewRoster && !$!isMyWorkspace) $tlang.getFormattedMessage("sinfo.manage.participants.moved","?sakai_action=doMenu_siteInfo_manageParticipants")
#end</div>`